tests/llmcompressor/transformers/obcq/obcq_configs/completion/gpu/llama_7b_quant.yaml

```diff
@@ -3,6 +3,6 @@ test_type: "regression"
 model: "meta-llama/Llama-2-7b-hf"
 dataset: open_platypus
 recipe: "tests/llmcompressor/transformers/obcq/recipes/quant.yaml"
-device: "auto"
+device: "cuda:0"
```

Comment:
What was the error you were getting with auto? I think we want at least one test confirming that auto is functional

Reply:
"Unspecified CUDA launch errors" - this was on an a100